wp-modules/editor/tests/EditorTest.php

```diff
@@ -54,11 +72,23 @@ public function test_register_pattern_post_type_meta_types() {
 	public function test_register_pattern_post_type_meta_defaults() {
 		register_pattern_post_type();
 
-		foreach ( array_diff( get_pattern_defaults(), [ 'title' => null ] ) as $meta_key => $default_value ) {
```

Comment:
The reason I had to change this was because it was failing with the error `Fatal error: Array to string conversion`. This is apparently the error message if you use `array_diff` with a multi-dimentional array. See:
https://stackoverflow.com/questions/19830585/array-to-string-conversion-error-when-calling-array-diff-assoc-with-a-multid

See lines 41 and 57:
<img width="723" alt="Screenshot 2023-05-19 at 4 41 44 PM" src="https://github.com/studiopress/pattern-manager/assets/7538525/d72d026c-6998-49f8-a07f-adc11dfc97a5">

I broke it out to be a bit easier for me to understand as well. I had a hard time fully understanding what was happening without breaking it out a bit.

Reply:
Thanks for your patience with this.

My obsession with terse code probably cost us a few hours in total.